ethminer/CMakeLists.txt

```diff
@@ -28,6 +28,32 @@ if (JSONRPC)
 	endif()
 endif()
 
+if (ETH_JSON_RPC_STUB)
```

Comment:
rather than copy'n'pasting all this code all over the cmake codebase, can't we just have a simple to use function?